src/go/k8s/tests/e2e/produce-consume/02-produce-message.yaml

```diff
@@ -19,9 +19,10 @@ spec:
           args:
             - >
               echo {"test":"message"} |
-              rpk topic produce test --brokers cluster-sample-0.cluster-sample.$POD_NAMESPACE.svc.cluster.local:9092
+              rpk topic produce test
+              --brokers cluster-sample-0.cluster-sample.$POD_NAMESPACE.svc.cluster.local:9092;cluster-sample-1.cluster-sample.$POD_NAMESPACE.svc.cluster.local:9092;cluster-sample-2.cluster-sample.$POD_NAMESPACE.svc.cluster.local:9092
```

Comment:
These semicolons should be commas, I think:

```
2021-09-17T11:29:38.707470219Z stdout F Reading message... Press CTRL + D to send, CTRL + C to cancel.
2021-09-17T11:29:44.967203679Z stderr F Error: kafka server: Request was for a topic or partition that does not exist on this broker.
2021-09-17T11:29:44.967244317Z stdout F Failed to send record
2021-09-17T11:29:44.968398278Z stderr F /bin/bash: cluster-sample-1.cluster-sample.kuttl-test-select-cobra.svc.cluster.local:9092: command not found
2021-09-17T11:29:44.968723355Z stderr F /bin/bash: cluster-sample-2.cluster-sample.kuttl-test-select-cobra.svc.cluster.local:9092: command not found
```

Reply:
You right fixed